src/SFA.DAS.ApprenticeCommitments.Web/Pages/Apprenticeships/ConfirmYourApprenticeshipDetails.cshtml

```diff
@@ -70,14 +70,16 @@
             The estimated duration of your apprenticeship may change during your apprenticeship after discussions with your employer and training provider.
         </div>
 
-        <form confirmation-form asp-show="@Model.ShowForm" method="post">
-            <yes-no-header>
-                Are your apprenticeship details correct?
-            </yes-no-header>
-
+        <form asp-show="@Model.ShowForm" method="post">
             <div class="govuk-form-group">
-                <span asp-validation-for="Confirmed" class="govuk-error-message"></span>
-                <yes-no-inputs asp-for="Confirmed"></yes-no-inputs>
+                <div validation-row-status property-name="Confirmed">
```

Comment:
Just curious really, does the formatting work if `validation-row-status` is put on the `div` above without creating a nested `div`?